src/data/matrix/basic.lean

```diff
@@ -60,6 +60,11 @@ by { ext, simp, }
 def transpose (M : matrix m n α) : matrix n m α
 | x y := M y x
 
+/-- The conjugate transpose of a matrix. -/
+def conj_transpose [has_star α] (M : matrix m n α) : matrix n m α
+| x y := star (M y x)
+localized "postfix `ᴴ`:1500 := matrix.conj_transpose" in matrix
+
 localized "postfix `ᵀ`:1500 := matrix.transpose" in matrix
```

Comment:
```suggestion
localized "postfix `ᵀ`:1500 := matrix.transpose" in matrix

/-- The conjugate transpose of a matrix. -/
def conj_transpose [has_star α] (M : matrix m n α) : matrix n m α :=
M.transpose.map star

localized "postfix `ᴴ`:1500 := matrix.conj_transpose" in matrix
```

Reply:
I did not see the advantage of defining this in this way. The first def seems more direct to me.

Reply:
It means you can use all the existing lemmas about `transpose` and `map`.

Reply:
I will change the order of the notation defs for sure.

Reply:
Make sense, I will do the change.